CMakeLists.txt

```diff
@@ -62,5 +62,6 @@ if(NBL_BUILD_EXAMPLES)
 	add_subdirectory(61_UI EXCLUDE_FROM_ALL)
 	add_subdirectory(62_CAD EXCLUDE_FROM_ALL)
 	add_subdirectory(62_SchusslerTest EXCLUDE_FROM_ALL)
+	add_subdirectory(64_MergeSort EXCLUDE_FROM_ALL)
```

Comment:
you can take nr 11 if you want

Reply:
This has been done, merge sort is now in example 11 over example 64.